Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1198,25 +1198,11 @@ private IEnumerable<string> GetUrlPrefixes()
 
         private CertificateInfo GetCertificateInfo(bool generateCertificate)
```

Comment:
I'm not 100% on the syntax
```suggestion
        private CertificateInfo GetCertificateInfo() => new CertificateInfo
        {
            Path = ServerConfigurationManager.Configuration.CertificatePath,
            Password = ServerConfigurationManager.Configuration.CertificatePassword
        };
```
cc @Bond-009

Reply:
I'm not sure either, I'd say leave it like this, we'll find out if the analyzers don't like it.